openstack/smn/v2/logtank/requests.go

```diff
@@ -0,0 +1,64 @@
+package logtank
+
+import (
+	"github.com/chnsz/golangsdk"
+)
+
+var RequestOpts = golangsdk.RequestOpts{
+	MoreHeaders: map[string]string{"Content-Type": "application/json;charset=UTF-8"},
+}
+
+// Ops is a struct that contains all the parameters.
+type Ops struct {
+	LogGroupID string `json:"log_group_id" required:"true"`
+
+	LogStreamId string `json:"log_stream_id" required:"true"`
```

Comment:
should use same style with LogGroupID
```suggestion
	LogStreamID string `json:"log_stream_id" required:"true"`
```